src/proto/ast.proto

```diff
@@ -67,6 +67,9 @@ message Declaration {
 	optional ChangeKind structural_change_kind = 10;
 	/** Kind of change on the label: UNCHANGED, ADDED, DELETED/REMOVED, RENAMED */
 	optional ChangeKind label_change_kind = 11;
+	/** For Specs use */
+	optional SpecStatement specs = 12;
```

Comment:
should this be 'repeated'?  Or is there exactly 0 or 1 specification statement per declaration?

Reply:
based on the naming of 'specs' (plural) I think we intended repeated here.

Reply:
Done.